listenbrainz/webserver/templates/index/listens_offline.html

```diff
@@ -0,0 +1,17 @@
+{%- extends 'base.html' -%}
+{%- block title -%}Listens currently unavailable - ListenBrainz{%- endblock -%}
+{%- block content -%}
+  <h2 class="page-title">Listens currently unavailable</h2>
+
+  <p>
+     The database that contains listens for our users is currently offline
+     for maintenance. Please try again in a few minutes.
```

Comment:
message saying that incoming listens are still being stored?

Reply:
On it.